packages/core/src/button/useButton.ts

```diff
@@ -82,7 +82,7 @@ export const useButton = <T extends Element>({
   };
 
   const buttonProps = {
-    "aria-disabled": disabled,
+    "aria-disabled": disabled && focusableWhenDisabled,
```

Comment:
```suggestion
    "aria-disabled": disabled && focusableWhenDisabled ? true : undefined,
```
Sorry, I think this should be this. This should prevent aria-disabled being applied when the button isn't disabled